sdk/eventhubs/azure-messaging-eventhubs/test/ut/processor_test.cpp

```diff
@@ -212,6 +214,9 @@ namespace Azure { namespace Messaging { namespace EventHubs { namespace Test {
 
         ProcessEventsForLoadBalancerTestSingleThreaded(producerClient, partitionClient, context);
       }
+
+      producerClient.Close(context);
```

Comment:
If we throw in this method won't we fail to close the client? Should the producer client be in a holder that will always call Close.

Ditto for the stop.

Reply:
If we throw in the method, the test will immediately stop, so closing the client isn't a big deal.

The only reason to have the explicit call to close here is to ensure that if the call to close *does* hang, the test fails within 5 minutes rather than waiting for 2 hours for the CI pipeline to figure out that it's been too long.